packages/instantsearch-ui-components/src/components/autocomplete/AutocompleteIndex.tsx

```diff
@@ -33,10 +38,21 @@ export type AutocompleteIndexClassNames = {
 
 export function createAutocompleteIndexComponent({ createElement }: Renderer) {
   return function AutocompleteIndex(userProps: AutocompleteIndexProps) {
-    const { items, ItemComponent, getItemProps, classNames = {} } = userProps;
+    const {
+      items,
+      HeaderComponent,
+      ItemComponent,
+      getItemProps,
+      classNames = {},
+    } = userProps;
 
     return (
       <div className={cx('ais-AutocompleteIndex', classNames.root)}>
+        {HeaderComponent && (
+          <div className={cx('ais-AutocompleteIndexHeader', classNames.header)}>
```

Comment:
Since there's no default header, the class seems a bit redundant to me. This would likely be something the user handles in their component no?
Are there some default styles that we have in the css?

Reply:
This will allow us to provide basic layout styles similar to https://github.com/algolia/autocomplete/blob/next/packages/autocomplete-theme-classic/src/theme.scss#L437.

I'll update the PR with those styles once design is merged.